packages/expect/src/__tests__/assertionCounts.test.ts

```diff
@@ -6,8 +6,6 @@
  *
  */
 
-'use strict';
-
 const {alignedAnsiStyleSerializer} = require('@jest/test-utils');
```

Comment:
can we use `import` in these tests instead of `require`?

Reply:
Fixed